checker/src/main/java/org/checkerframework/checker/determinism/DeterminismVisitor.java

```diff
@@ -611,7 +560,7 @@ private boolean isSubtypeError(
      * @param errorMessage the error message to report
      * @return true if {@code elementAnno} is a subtype of {@code supperAnnotation} and it's not the
      *     case that {@code collectionAnno} is {@code @NonDet} and {@code elementAnno} is
-     *     {@code @Det} or {@code @OrderNonDet}, false otherwise
```

Comment:
The Javadoc above refers to `subAnnotation` and `supperAnnotation`, but those aren't the names of the parameters.